selfdrive/car/gm/carstate.py

```diff
@@ -62,14 +60,13 @@ def update(self, pt_cp, loopback_cp):
     ret.rightBlinker = pt_cp.vl["BCMTurnSignals"]["TurnSignals"] == 2
 
     self.park_brake = pt_cp.vl["EPBStatus"]["EPBClosed"]
-    ret.cruiseState.available = bool(pt_cp.vl["ECMEngineStatus"]["CruiseMainOn"])
+    ret.cruiseState.available = pt_cp.vl["ECMEngineStatus"]["CruiseMainOn"] != 0
```

Comment:
IMHO this and other similar changes are a loss of clarity. `bool()` makes it clear at a glance the underlying data is expected to be true/false, whereas `!= 0` could be applied to any numeric input. We deal with lots of 2/3/4-bit wide CAN integers.

Reply:
merging since I'd like to get this in. I don't have a strong opinion on this. We'll take a look at this a bit later as we clean the car code up.